### ISSUE TITLE
StartMetricsExporter breaks LastValue aggregation

### ISSUE THREAD

**Issue by jkohen** [census-ecosystem/opencensus-go-exporter-stackdriver#242]
Please answer these questions before submitting a bug report.

### What version of the Exporter are you using?
v0.12.8

### What version of OpenCensus are you using?
v0.22.2

### What version of Go are you using?
go version go1.13.4 linux/amd64

### What did you do?
Following https://godoc.org/contrib.go.opencensus.io/exporter/stackdriver#Exporter.StartMetricsExporter, I replaced:
```go
			sd, _ := oc_stackdriver.NewExporter(oc_stackdriver.Options{
				ProjectID: *projectId,
				ReportingInterval: reportingInterval,
			})
			defer sd.Flush()
			view.RegisterExporter(sd)
			view.SetReportingPeriod(reportingInterval)
```

with:
```go
			sd, _ := oc_stackdriver.NewExporter(oc_stackdriver.Options{
				ProjectID: *projectId,
				ReportingInterval: reportingInterval,
			})
			exporter.StartMetricsExporter()
			defer exporter.StopMetricsExporter()
```

### What did you expect to see?
All my existing metrics continue to show up.

### What did you see instead?
A gauge defined as follows stopped showing new points in Stackdriver. Other metrics continued to work as before. Namely, the ones from `ocgrpc` and `ochttp`.
```go
	// VersionTag identifies the version of this binary.
	VersionTag = tag.MustNewKey("version")
	// UptimeMeasure is a metric.
	UptimeMeasure = stats.Int64("agent.googleapis.com/agent/uptime", "uptime of the Stackdriver Prometheus collector", stats.UnitSeconds)

		&view.View{
			Measure:     UptimeMeasure,
			TagKeys:     []tag.Key{VersionTag},
			Aggregation: view.LastValue(),
		},
```

**Comment by rghetia** [comment]
@jkohen I tried this [example](https://github.com/census-ecosystem/opencensus-go-exporter-stackdriver/blob/master/examples/stats/main.go) after replacing Aggregation.Distribution with Aggregation.LastValue(). It is working for me.

**Comment by eqinox76** [comment]
I had a similar issue where no new custom metrics were shown in stackdriver. Following
[stackoverflow](https://stackoverflow.com/questions/56937653/custom-opencensus-metrics-not-appearing-on-stackdriver) I added:
```go-lang
view.RegisterExporter(sd)
view.SetReportingPeriod(reportingInterval)
```
Now its working fine for me.

**Comment by rghetia** [comment]
@eqinox76 and @jkohen   both method works.
old way uses [ExportView()](https://github.com/census-ecosystem/opencensus-go-exporter-stackdriver/blob/1cdca918b59d4cd456dc1bbfb4b655159da9a62e/stackdriver.go#L381)
```
view.RegisterExporter(sd)
view.SetReportingPeriod(reportingInterval)
```
new way uses [ExportMetrics()](https://github.com/census-ecosystem/opencensus-go-exporter-stackdriver/blob/1cdca918b59d4cd456dc1bbfb4b655159da9a62e/stackdriver.go#L398)
```
se.StartMetricsExporter()
```
@jkohen I tried using old way first and then use the new way for the same metric (Gauge/LastValue). It works.

Can you reproduce this?

**Comment by jkohen** [comment]
Sorry, Rahul, I no longer have the code and environment to test. I ended up dropping the gauges and tore down the test environment. This isn't blocking us, so if we don't have a way to reproduce this, we can close the issue.

**Comment by gonzaloserrano** [comment]
Hey people,

Somehow in our code we had the old system and the new system and kept seeing errors in our logs. We switched to the new one and stopped receiving custom metrics in Stackdriver. Now I switched to the old one and can see them again in Stackdriver but the errors are still there (`stackdriver error: rpc error: code = InvalidArgument desc = One or more TimeSeries could not be written: Points must be written in order`)

Any hints?
